### DISCUSSION TITLE
model memory leak profiling

### DISCUSSION THREAD

**Discussion by ghost** [triton-inference-server/server#6854] [Q&A]
How do you guys profile the memory usage of the deployed model?

I am deploying a model using triton serving, but turns out the host memory increases as the number of requests increases during the stress test. I am trying to figure out what part of my model occurs the leak and seeing [this document](https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/user_guide/debugging_guide.html?highlight=memory#performance-profiling). 
I tried with nsys and capture the nsys report, but I couldn't find the memory heap stacks in the report...

Thank you in advance.

**Comment by ghost** [comment]
Found out [this issue](https://github.com/triton-inference-server/server/issues/6781) and it seems that the contributors are used to check the possible leaks with this tool. I will go with this tool.

**Reply by sboudouk** [reply]
Did you succeed into profiling & identifying your model leak ?
